Functions.Templates/Templates/SqlInputBinding-Python/function.json

```diff
@@ -0,0 +1,26 @@
+{
+    "scriptFile": "__init__.py",
+    "bindings": [
+      {
+        "type": "httpTrigger",
+        "direction": "in",
+        "name": "req",
+        "authLevel": "anonymous",
+        "route": "getproducts/{cost}"
```

Comment:
We should remove the route - the user will set the name during creation so that should be the default route